packages/lib-sourcify/src/lib/CheckedContract.ts

```diff
@@ -530,6 +538,18 @@ export class CheckedContract {
       throw error;
     }
   }
+
+  exportConstructorArguments() {
+    return {
+      metadata: this.metadata,
+      solidity: this.solidity,
+      missing: this.missing,
+      invalid: this.invalid,
+      creationBytecode: this.creationBytecode,
```

Comment:
Since we removed create2 features we can also remove the `creationBytecode` from the session